src/encoded/static/components/app.js

```diff
@@ -322,8 +321,10 @@ class App extends React.Component {
             logout: 'triggerLogout',
         };
 
+        this.href = props.href;
```

Comment:
Since we are using `this.href` here, do we need `this.state.href` set in line 301?